### PR TITLE
Handle 2022 polls in model

### PR DESCRIPTION
**Pull request by AlexAndorra**
READY TO MERGE 🍾 

This PR adds / edits the functions as needed to integrate the [2022 polls](https://github.com/pollsposition/data/blob/main/sondages/presidentielles_2022.json) to the model.
Once this is merged, we'll be able to run the model on 2022 data 🥳 

@rlouf you can already review, to make the process easier and faster for you. I'll ping you each time a new portion can be reviewed. No need to review the notebook, just the python program.

TODO:
- [x] Cleanup up raw 2022 JSON polls
- [x] Check data processing code still works
- [x] Update unemployment data
- [x] Remove unused predictors

### CODE REVIEW

**Comment by rlouf** [comment]
You know there's a "draft PR" feature on github right?

**Comment by AlexAndorra** [comment]
Yeah but you would not be able to officially accept / reject my changes until I mark it as ready for review

**Comment by AlexAndorra** [comment]
Ready to review and merge @rlouf 🍾 
I tested and the model runs without issues on the new data

File: presidential-elections/notebooks/dynamic-model-final.ipynb [hunk 1]
```diff
@@ -3,12 +3,16 @@
   {
```
Comment (rlouf) [inline presidential-elections/notebooks/dynamic-model-final.ipynb:3]: <p>Mmm what is this doing here if you're processing and merging the 2022 elections below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pollsposition/models/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (rlouf) [inline presidential-elections/notebooks/dynamic-model-final.ipynb:3]: <p>:(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pollsposition/models/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by rlouf** [comment]
I couldn't see the predictions for 2022 ? :)

Looks good to me; this will need some refactoring in the near future but it works and we can start using it!

File: presidential-elections/notebooks/dynamic-model-final.ipynb (hunk 1 above)
Comment (AlexAndorra) [reply presidential-elections/notebooks/dynamic-model-final.ipynb:3]: <p>No need to review the NB, it's not ready. The code to review is the python script</p>

**Comment by rlouf** [comment]
Yep, python file did look good to me. You can go ahead and merge. I'll try to find some time for a refactor and documentation of the model later this week or next week :)

**Comment by AlexAndorra** [comment]
Yeah, because the NB is not ready. You only needed to review the python script, as I wrote above. 
Merging then, thanks for reviewing!

**Pull request merged**